terraform/aws/components/lab-results/module_ecs_service.tf

```diff
@@ -0,0 +1,57 @@
+module "lab-results_ecs_service" {
+  source = "../../modules/module_ecs_service"
+
+  project         = var.project
+  component       = var.component
+  environment     = var.environment
+  region          = var.region
+  module_instance = "lab-results_ecs"
+  default_tags    = local.default_tags
+
+  availability_zones = local.availability_zones
+
+  image_name        = local.image_name
+  cluster_id        = data.terraform_remote_state.base.outputs.base_cluster_id
+  minimal_count     = var.lab-results_service_minimal_count
+  desired_count     = var.lab-results_service_desired_count
+  maximal_count     = var.lab-results_service_maximal_count
+  service_target_request_count = var.lab-results_service_target_request_count
+
+  container_port    = var.lab-results_service_container_port
+  application_port  = var.lab-results_service_application_port
+  launch_type       = var.lab-results_service_launch_type
+  log_stream_prefix = var.lab-results_build_id
+  healthcheck_path  = var.lab-results_healthcheck_path
+  enable_load_balancing = true
+
+  container_healthcheck_port = var.lab-results_service_container_port
+  enable_dlt                 = var.enable_dlt
+  dlt_vpc_id                 = var.dlt_vpc_id
+
+  environment_variables = local.environment_variables
+  secret_variables      = local.secret_variables
+
+  task_execution_role_arn = aws_iam_role.ecs_service_task_execution_role.arn
+  task_role_arn           = data.aws_iam_role.ecs_service_task_role.arn
+  task_scaling_role_arn   = data.aws_iam_role.ecs_autoscale_role.arn
+
+  additional_security_groups = [
+    data.terraform_remote_state.base.outputs.core_sg_id,
+    data.terraform_remote_state.base.outputs.docdb_access_sg_id,
+    aws_security_group.lab-results_external_access.id
+  ]
+
+  lb_allowed_security_groups = [
+    data.terraform_remote_state.account.outputs.jumpbox_sg_id
+  ]
+
+  container_allowed_security_groups =  [
+    data.terraform_remote_state.account.outputs.jumpbox_sg_id,
```

Comment:
this is only required if the LB is a network type, for application type, access via LB is suffiecient

Reply:
above lines are comment out in fe43b60e5d0e25d85cfea4eb4f23d24dab5b0081